lib/phantomscript.js

```diff
@@ -75,7 +75,7 @@ files.forEach(function (file) {
   var oDOM
   var svgContent
 
-  console.log('ready to execute png: ' + filename + '.png ')
+  log('ready to execute: ' + file)
```

Comment:
Note that this change makes execution completely silent without `--verbose`. If that's undesirable, we _could_ use `console.log` for "ready to execute" and "saved" messages.